src/app/services/debug.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import {HttpClient, HttpHeaders} from '@angular/common/http';
+import { environment } from '@env/environment';
+import { map } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DebugService {
+
+  constructor(private http: HttpClient) { }
+
+  getList() {
+    return this.http.get<any>(`${environment.apiUrl}/debug`)
```

Comment:
Should I return a response object, such as ApiResponse?

Reply:
This would be useful, in case if we need some additional fields for pagination, i.e. totalRecors, раgeSize and so on.
I would leave it as is for now, and get back to this discussion later.